pkg/daemon/server/service/pipeline_watermark_query.go

```diff
@@ -68,7 +60,7 @@ func (ps *pipelineMetadataQuery) GetPipelineWatermarks(ctx context.Context, requ
 		watermarkArr := make([]*daemon.EdgeWatermark, len(ps.watermarkFetchers))
 		i := 0
 		for k := range ps.watermarkFetchers {
-			edgeName := k
+			edgeName := k.From + "-" + k.To
```

Comment:
since `k` is `v1alpha1.Edge`, perhaps we should write a `Stringer` for `v1alpha1.Edge`?

Reply:
added that